src/interrogate/config.py

```diff
@@ -134,8 +138,7 @@ def parse_pyproject_toml(path_config: str) -> dict[str, Any]:
     :raise OSError: an I/O-related error when opening ``pyproject.toml``.
     :raise tomllib.TOMLDecodeError: unable to load ``pyproject.toml``.
     """
-    with open(path_config, "rb") as f:
-        pyproject_toml = tomllib.load(f)
+    pyproject_toml = tomllib.loads(Path(path_config).read_text())
```

Comment:
```suggestion
    pyproject_toml = tomllib.loads(Path(path_config).read_text(encoding="utf-8"))
```
No encoding is specified.